samples/subsys/mgmt/mcumgr/smp_svr/sample.yaml

```diff
@@ -12,9 +12,11 @@ tests:
       - nrf52840dk/nrf52840
       - pinnacle_100_dvk
       - mg100
+      - nrf54l15pdk/nrf54l15/cpuapp
     integration_platforms:
       - nrf52dk/nrf52832
       - nrf52840dk/nrf52840
+      - nrf54l15pdk/nrf54l15/cpuapp
```

Comment:
So this should go to sdk-zephyr as a no-up since bluetooth is not supported for this board in zephyr